File: 782-jewels-and-stones/jewels-and-stones.go
func numJewelsInStones(jewels string, stones string) int {
    var umap = make(map[rune]int);
    for _, i := range jewels {
        umap[i] = 0
    }
    count := 0;
    for _, i := range stones {
        _ , tmp := umap[i]
        if tmp{
            umap[i]++;
            count++;
        }
    }
    return count;
}

File: 782-jewels-and-stones/jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        umap = {ch:0 for ch in jewels};
        for i in stones:
            if i in umap:
                umap[i] += 1;
        return sum(umap.values());


        